packages/react-tc-ui-kit/src/stories/UIKit.stories.js

```diff
@@ -56,9 +56,13 @@ export const InputExample = () => {
 
       <Input
         type={text('inputType', 'text')}
+        label={text('labelText', 'label')}
         placeholder=""
         value={inputValue}
         disabled={boolean('disabled', false)}
+        isInvalid={boolean('isInvalid', false)}
+        validationMessages={[text('validationMessages', 'validationMessages')]}
```

Comment:
i think we should provide more readable default string (2nd argument)\
for example `Validation Message`